doc/developers/contributing.rst

```diff
@@ -952,6 +952,18 @@ The easiest and recommended way to accomplish this is to
 All logic behind estimator parameters,
 like translating string arguments into functions, should be done in ``fit``.
 
+Parameters can be passed to the ``fit`` method. Generally, these parameters
+should be restricted to variables that have the shape ``n_samples``.
+This allows for all relevant parameters to be sliced cleanly during
+cross-validation. All other parameters should be passed to
+``__init__`` or ``set_params``.
```

Comment:
Oops, sorry I didn't put the underscores with init in a code block and it bolded. I didn't mean to suggest a change to remove underscores, that's good as it is